project_timesheet_hour/models/project_project.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class ProjectProject(models.Model):
+
+    _inherit = 'project.project'
+
+    planned_hours = fields.Float(
+        string="Planned hours",
+        compute='_compute_planned_hours',
+        store=True
+    )
+    total_hours_spent = fields.Float(
+        string="Total hours spent",
+        compute='_compute_spent_hours',
+        store=True
+    )
+    progress = fields.Float(
+        string="Progress",
+        compute='_compute_progress',
+        store=True
+    )
+
+    @api.multi
+    @api.depends(
+        'task_ids.planned_hours')
+    def _compute_planned_hours(self):
+        for rec in self:
+            rec.planned_hours = rec._get_planned_hours()
+
+    @api.multi
+    @api.depends(
+        'task_ids.total_hours_spent')
+    def _compute_spent_hours(self):
+        for rec in self:
+            rec.total_hours_spent = rec._get_total_hours_spent()
+
+    @api.multi
+    @api.depends(
+        'total_hours_spent', 'planned_hours')
+    def _compute_progress(self):
+        for rec in self:
+            rec.progress = rec._get_progress()
+
+    @api.multi
+    def _get_planned_hours(self):
```

Comment:
You don't need now this intermediate methods (that's why I said about renaming directly these methods to `_compute`). Any way, please include the code directly in `_compute` methods.

Reply:
HI @benwillig could you please check this?